build-image/Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN apt-get update && apt-get install -y \
   && rm -rf /var/lib/apt/lists/*
 
 # Install dependencies used for the operator
-RUN go install sigs.k8s.io/controller-tools/cmd/controller-gen@latest
```

Comment:
Someone/something probably needs to build/tag a new container here, and bump the version in the Makefile while merging this.